eng/common/scripts/Update-ChangeLog.ps1

```diff
@@ -36,6 +39,7 @@ if ($ReleaseDate)
     }
 }
 elseif ($Unreleased) {
+
```

Comment:
Did you intend to put the `{` on this line as well?

Reply:
Nope, Thanks.